app/src/main/res/values/strings.xml

```diff
@@ -520,5 +520,20 @@
     <string name="block_user_dialog_title">Block user?</string>
     <string name="block_user_dialog_body">%1$s won’t be able to contact you or add you to group conversations. You will lose access to any 1:1 conversations with them. The other user will not be notified about this.</string>
     <string name="block_user_dialog_confirm_button">Block</string>
+    <string name="general">GENERAL</string>
+    <string name="devices">DEVICES</string>
+    <string name="backups">BACKUPS</string>
+    <string name="others">OTHERS</string>
```

Comment:
I would put some more descriptive keys to these strings, to help later on identify where does this belong and of course to help translations or accessibility later on :D 

Also, I think we could use just normal lowercase strings, and convert it in place to the desired case.